ideas/101-extensions.md

```diff
@@ -71,4 +135,4 @@ Iterations will be defined based on specification and team feedback.
 - 3 external extensions shipped
```

Comment:
Maybe add something about usage and adoption of extensions?

Reply:
:+1: